plugin/manager.go

```diff
@@ -146,11 +153,11 @@ func (m *CLIManager) Install(ctx context.Context, installOpts CLIInstallOptions)
 	// validate and get new plugin metadata
 	newPlugin, err := NewCLIPlugin(ctx, pluginName, pluginExecutableFile)
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to create new CLI plugin: %w", err)
+		return nil, nil, err
 	}
 	newPluginMetadata, err := newPlugin.GetMetadata(ctx, &proto.GetMetadataRequest{})
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to get metadata of new plugin: %w", err)
+		return nil, nil, err
```

Comment:
The information `failed to get metadata of new plugin` is lost in this change. When installing a plugin that's already installed, user needs to know which plugin's `get-plugin-metadata` failed.

Reply:
Restored.